docs/Using_PWM_Tools.md

```diff
@@ -0,0 +1,73 @@
+This document describes how to setup a PWM-Controlled laser or spindle
+using `output_pin` and some macros.
```

Comment:
Generally new documents should also be added to docs/Overview.md (otherwise, no one is likely to find it).  Though I wonder if adding this info to something like config/sample-laser.cfg might be better?